.eslintrc.js

```diff
@@ -1,3 +1,16 @@
 module.exports = {
-  "extends": "google"
-};
\ No newline at end of file
+  'extends': 'google',
+  'parser': 'babel-eslint',
```

Comment:
try not to change the default parser, you can solve the es6 and jsx syntax with eslint

https://eslint.org/docs/user-guide/configuring#specifying-parser-options